api/catalog/api/models/media.py

```diff
@@ -70,19 +87,19 @@ class AbstractMedia(
     meta_data = models.JSONField(blank=True, null=True)
 
     @property
-    def license_url(self):
+    def license_url(self) -> str:
```

Comment:
The license_url should really be available in all media. I know catalog now requires every media item to have it, I wonder how many items in the current DB don't have the URL value...

Reply:
This is a computed field, so if the license name and version are present, the URL can be determined.

Reply:
I know, but the catalog is computing this field, too, if it doesn't get it from providers. And it now saves all three items: license name, version, and URL. So we shouldn't be doing computing here as well.

Reply:
The API database does not have this value as a model field, so the ingestion server must not be copying from the catalog. We should make a new issue to add this field to the model and then we can gradually retire this computed function to be a fallback for fields that don't have this value.

Reply:
Oh, I wasn't entirely correct.  The license_url *is* now saved, but it is saved as a property of metadata, and not as a separate column.